index.html

```diff
@@ -106,6 +107,76 @@ <h3>Small -</h3>
 &lt;/div&gt;
           </pre>
         </section>
+        <hr />
+        <section class="parent-section">
+          <h1>Navbar Component</h1>
+          <div class="navbar-container">
+            <nav class="navbar">
+              <div class="navbar-title">Neo Cart</div>
+              <div class="navbar-search-input-container">
+                <div class="input-lefticon">
+                  <img src="./assets/images/search-icon.svg" alt="" />
+                </div>
+                <input
+                  class="navbar-search-input"
+                  type="text"
+                  placeholder="Search"
+                />
+              </div>
+              <div class="navbar-right-options">
+                <button>Login</button>
+                <div>
+                  <img
+                    height="24px"
+                    src="./assets/images/Wishlist.svg"
+                    alt=""
+                  />
+                </div>
+                <div class="flex-row items-center">
+                  <img
+                    height="24px"
+                    src="./assets/images/Cart-Icon.svg"
+                    alt=""
+                  />
+                  <label>Cart</label>
+                </div>
+              </div>
+            </nav>
+          </div>
+          <pre class="avatar-code">
+            &lt;nav class=&quot;navbar&quot;&gt;
+              &lt;div class=&quot;navbar-title&quot;&gt;Neo Cart&lt;/div&gt;
+              &lt;div class=&quot;navbar-search-input-container&quot;&gt;
+                &lt;div class=&quot;input-lefticon&quot;&gt;
+                  &lt;img src=&quot;./assets/images/search-icon.svg&quot; alt=&quot;&quot; /&gt;
+                &lt;/div&gt;
+                &lt;input
+                  class=&quot;navbar-search-input&quot;
+                  type=&quot;text&quot;
+                  placeholder=&quot;Search&quot;
+                /&gt;
+              &lt;/div&gt;
+              &lt;div class=&quot;navbar-right-options&quot;&gt;
+                &lt;button&gt;Login&lt;/button&gt;
+                &lt;div&gt;
+                  &lt;img
+                    height=&quot;24px&quot;
+                    src=&quot;./assets/images/Wishlist.svg&quot;
+                    alt=&quot;&quot;
+                  /&gt;
+                &lt;/div&gt;
+                &lt;div class=&quot;flex-row items-center&quot;&gt;
+                  &lt;img
+                    height=&quot;24px&quot;
+                    src=&quot;./assets/images/Cart-Icon.svg&quot;
+                    alt=&quot;&quot;
+                  /&gt;
+                  &lt;label&gt;Cart&lt;/label&gt;
+                &lt;/div&gt;
+              &lt;/div&gt;
+            &lt;/nav&gt;
+          </pre>
+        </section>
       </div>
     </div>
     <script src="app.js"></script>
```

Comment:
Suggestion: Add this script inside last line of `<head>` tag and add `defer` attribute to it.
It will improve the performance.
like: 
```
<script src="app.js" defer></script>
```